app/email/evidence-email.js

```diff
@@ -15,9 +15,9 @@ const REVIEW_BULLET_POINTS_BY_TYPE_OF_LIVESTOCK = {
   [SHEEP]: REVIEW_SHEEP
 }
 
-const getFollowUpBulletPoints = (typeOfLivestock, testResults, piHuntRecommended) => {
+const getFollowUpBulletPoints = (typeOfLivestock, reviewTestResults, piHuntRecommended) => {
   if ([BEEF, DAIRY].includes(typeOfLivestock)) {
-    if (testResults === 'positive') { return FOLLOW_UP_CATTLE_POSITIVE }
+    if (reviewTestResults === 'positive') { return FOLLOW_UP_CATTLE_POSITIVE }
     if (piHuntRecommended === 'yes') { return FOLLOW_UP_CATTLE_NEGATIVE_RECOMMENDED_PI_HUNT }
```

Comment:
I think this needs to consider another field. 
If review test results negative **and** vet recommended pi Hunt **and** pi hunt all animals, then this applies